src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery[] lotteryArray = {new Lottery(), new Lottery(), new Lottery()};
```

Comment:
you need just one object lottery. His method getRandomBall() return new Ball each time